analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/yarn-workspaces-expected-output.yml

```diff
@@ -0,0 +1,181 @@
+---
+project:
+  id: "Yarn::workspace:1.0.0"
+  definition_file_path: "analyzer/src/funTest/assets/projects/synthetic/yarn-workspaces/package.json"
+  declared_licenses:
+  - "ISC"
+  declared_licenses_processed:
+    spdx_expression: "ISC"
```

Comment:
This needs to be updated to Apache-2.0 now as well.

Reply:
Done.